sites/fast-creator/app/preview.tsx

```diff
@@ -168,14 +169,35 @@ class Preview extends Foundation<{}, {}, PreviewState> {
                 "style",
                 "background: var(--background-color); height: 100%;"
             );
+        }
+    }
 
+    private attachComponentsAndInit(): void {
+        this.attachMappedComponents();
+        if (this.state.dataDictionary !== undefined) {
             this.state.htmlRenderMessageSystem.postMessage({
                 type: MessageSystemType.initialize,
                 dataDictionary: this.state.dataDictionary,
                 schemaDictionary: this.state.schemaDictionary,
             });
+            if (this.state.activeDictionaryId) {
+                this.state.htmlRenderMessageSystem.postMessage({
+                    type: MessageSystemType.navigation,
+                    action: MessageSystemNavigationTypeAction.update,
+                    activeDictionaryId: this.state.activeDictionaryId,
+                    options: {
+                        originatorId: "preview",
+                    },
+                    activeNavigationConfigId: "",
+                });
+            }
         }
     }
+
+    private attachComponentsAndData(): void {
```

Comment:
Not sure I'm seeing where this is getting used.

Reply:
It wasn't. Dead code that I failed to clean up. It has been removed.